webpack-common.js

```diff
@@ -146,6 +145,13 @@ export function getPlugins({ excludeOtherAppLocales = true } = {}) {
     }),
   ];
 
+  if (includeLoadablePlugin) {
+    // We need this file to be written on disk so that our server code can read
+    // it. In development mode, webpack usually serves the file from memory but
+    // that's not what we want for this file.
+    plugins.push(new LoadablePlugin({ writeToDisk: true }));
```

Comment:
I believe this plugin should be listed first, but maybe not anymore.

Reply:
thanks @willdurand. So would `unshift` work (instead of `push`)? or are you saying this is okay how it is? I am not clear..

Reply:
check out the loadable docs in case it's written. maybe it's not a requirement anymore.

Reply:
I don't see anything about this..

Reply:
then I suppose you're good to go